crates/types/src/config/admin.rs

```diff
@@ -80,6 +95,9 @@ impl Default for AdminOptions {
             #[cfg(any(test, feature = "test-util"))]
             data_dir: super::default_arc_tmp(),
             heartbeat_interval: Duration::from_millis(1500).into(),
+            // try to trim the log every hour
+            log_trim_interval: Some(Duration::from_secs(60 * 60).into()),
+            log_trim_threshold: 1000,
```

Comment:
I'm not sure having a delta that's calculated by the "number" of lsns really make sense in the long term. The number of LSNs can't reflect whether a record is durable or not, but I'm okay with leaving this until we have a mechanism to offset lsns by time of creation instead (i.e. "older than X")